### ISSUE TITLE
Include alt text (and other attributes like title text)?

### ISSUE THREAD

**Issue by missmobtown** [backdrop-contrib/file_viewmode_token#1]
Hello! I was really happy to find this module, as it helped me solve the problem I ran into during migration: in my D7 site, images had been inserted in the WYSIWYG field using WYSIWYG Media Browser, and after migration the Backdrop site was treating those embedded images as text.

Once I got the correct view modes added to the Backdrop site, your module fixed the output on the front end. There's one thing missing that I hoped could be added: ALT text. Here is the output showing in inspector; it looks like it wants to include alt text but comes out empty?
```
<div class="content">
          <img src="https://website.com/files/image.gif" width="350" height="453" alt="">
</div>

```
